### ISSUE TITLE
Dungeon not staying sealed

### ISSUE THREAD

**Issue by Fenyx4** [cambragol/advent-of-the-trinity#166]
I assume this is a bug but I'm not sure.

I sealed Wrong. Then a hydra shot at me at hit the entrance blowing it open again.

I hope sealing it multiple times doesn't cause me issues. :)

**Comment by cambragol** [comment]
Wow.  That is crazy.

Let me think on this state for a moment. It might work out, as the words of power just check the current tile status, then modify accordingly.

So it can accomodate the entrance being blow open, then sealed again. So...a feature!!  Maybe.

**Comment by cambragol** [comment]
Yes, this is a feature.  Trying yelling the word of power again, and you can open and close the dungeons. It is all built in. 

It is kind of cool that the hydra was able to do that. Emergent gameplay!

**Comment by Fenyx4** [comment]
Lol. Awesome!!

**Comment by cambragol** [comment]
There is an issue with this...

The sealing of the Crypts is more involved than the dungeons, and results in a final state where the entrance is 'permanently' closed (It is a mountain)

However, a ship, hydra, fire lizard, dragon or balron blast could destroy the mountain, and reveal the 'closed cave' icon. 

Nothing major, but the crypt could then be reopened, and reentered. 

Hmph.

**Comment by cambragol** [comment]
Fixed. Mountains are now indestructable. But I made balloons destructable in their stead, to converse total code. I figure it is easy enough to get another balloon by just exiting Hythloth.

Besides, balloons are very destructable. 

Hopefully that doesn't raise an issue while riding it...default U4 freezes all monsters while the balloon is flying. Trinity does not, I think. Which means a stray fireball could take you out...what would happen..?
